src/todolist/settings.py

```
@@ -18,7 +18,7 @@
 # See https://docs.djangoproject.com/en/1.7/howto/deployment/checklist/
 
 # SECURITY WARNING: keep the secret key used in production secret!
-SECRET_KEY = "@e2(yx)v&tgh3_s=0yja-i!dpebxsz^dg47x)-k&kq_3zf*9e*"
+SECRET_KEY = os.getenv("SECRET_KEY")
```

This line uses `os.getenv("SECRET_KEY")`, which returns `None` silently if the environment variable is not set. The task requires reading `SECRET_KEY` from the environment at runtime in a way that fails clearly if not present (e.g., `os.environ['SECRET_KEY']` or raising an explicit error). Please change this to either:

- Use `SECRET_KEY = os.environ['SECRET_KEY']` so Python raises `KeyError` if missing, or
- Use a clearer failure with Django's `ImproperlyConfigured`, for example:

```python
from django.core.exceptions import ImproperlyConfigured
SECRET_KEY = os.environ.get('SECRET_KEY')
if not SECRET_KEY:
    raise ImproperlyConfigured('SECRET_KEY environment variable is not set')
```

This will satisfy the checklist requirement that the app uses the SECRET_KEY from the environment and fails loudly if it is not provided.